### PR TITLE
addListener starts emit process like `on` method

### PR DESCRIPTION
**Pull request by ZigGreen**
resolves: #214

### CODE REVIEW

File: test/issues.test.js [hunk 1]
```diff
@@ -300,4 +300,20 @@ it.describe("github issues", function (it) {
               });
         });
     });
+
+    it.describe("#214", function (it) {
+        it.timeout(100);
+
+        it.should("emmit data when on called", function (next) {
```
Comment (dustinsmith1024) [inline test/issues.test.js]: i think its emit with 1 m? :P

**Comment by dustinsmith1024** [comment]
Thanks for the PR. I'll wait a day or so for feedback from anyone else.

@doug-martin any thoughts/concerns?

File: test/issues.test.js (hunk 1 above)
Comment (ZigGreen) [reply test/issues.test.js]: whoops!

File: test/issues.test.js [hunk 2]
```diff
@@ -300,4 +300,20 @@ it.describe("github issues", function (it) {
               });
         });
     });
+
+    it.describe("#214", function (it) {
+        it.timeout(100);
+
+        it.should("emit data when on called", function (next) {
+            stream = csv.fromPath(path.resolve(__dirname, "./assets/test1.csv"));
```
Comment (doug-martin) [inline test/issues.test.js]: Add a var when declaring the stream in these two tests.

Comment (ZigGreen) [reply test/issues.test.js]: done :)

**Comment by doug-martin** [comment]
This was fixed in v3 but I ported your tests over just to make sure there are no regressions. https://github.com/C2FO/fast-csv/commit/5274a96173889b151a7c84e8c5038f3b3ce118a6

**Pull request closed without merging**